.github/scripts/bump-kubectl-patch-versions

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Note for reviewers: this script is based off one we already use in `rancher/kubectl` repo.